packages/user-interface/src/components/MenuItem.tsx

```diff
@@ -15,7 +16,15 @@ interface MenuItemProps {
 
 const MenuItem = ({ item, current = false }: MenuItemProps) => {
   const { hrefLang } = useContext(LocaleContext);
-  const { hideMenu, messagesCount } = useContext(LayoutContext);
+  const { hideMenu, messagesCount, setMessagesCount } =
```

Comment:
Is this messagesCount used somewhere else? Otherwise we can remove it from this context

Reply:
MenuItem changes were not supposed to be part of this PR. Sorry. Will revert.

Those will come in the next PR I'm working on.